### PR TITLE
init LG app submission review

### PR DESCRIPTION
**Pull request by lstod**
Populated known parts of application_submission_review from the Application_Local_Government sheet
- Missing created_by_uuid and is_authorized

### CODE REVIEW

File: bin/migrate-oats-data/applications/migrate_application.py [hunk 1]
```diff
@@ -32,11 +34,13 @@ def process_application_etl(batch_size):
     process_application_owners(batch_size)
     process_app_staff_journal(batch_size)
     process_application_decisions(batch_size)
+    insert_application_submission_review(batch_size)
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/migrate_application.py:37]: Could you move this closer to submission processing for readability consistency

**Review by mhuseinov — APPROVED** [review]
👍

File: bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        "local_government_file_number": row["local_gov_file_number"],
+        "audit_created_by": OATS_ETL_USER,
+    }
+
+
+
+
+def _map_ocp_designation(ocp_value):
+
+    if ocp_value["community_pln_compliance_ind"] == 'X':
```
Comment (trslater) [inline bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py:147]: Is this meant to be 'Y'?

File: bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    if ocp_value["community_pln_compliance_ind"] == 'N':
+        return False
+    elif ocp_value["community_pln_compliance_ind"] == 'Y':
+        return True
+    else:
+        return None
+
+def _map_zoning(zoning_value):
+ 
+    if zoning_value["zoning_compliance_ind"] == 'X':
```
Comment (trslater) [inline bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py:165]: Same as above.

File: bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py (hunk 2 above)
Comment (lstod) [reply bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py:147]: You would think that but no, that is the correct mapping. 
`if community_pln_designation = X -> false , if community_pln_designation = N -> true`

Comment (trslater) [reply bin/migrate-oats-data/applications/submissions/review/lg_submission_review.py:147]: Weird...

**Review by trslater — APPROVED** [review]

**Pull request merged**